src/nvim/ex_cmds.c

```diff
@@ -5016,7 +5016,9 @@ helptags_one (
     /*
      * Sort the tags.
      */
-    sort_strings((char_u **)ga.ga_data, ga.ga_len);
+    if (ga.ga_data != NULL) {
```

Comment:
Not really needed, but coverity does not know about `ga.ga_len` that can be checked.